components/table/TableRow.js

```diff
@@ -60,6 +60,9 @@ const factory = (Checkbox) => {
     renderInput (key, value) {
       const index = this.props.index;
       const inputType = utils.inputTypeForPrototype(this.props.model[key].type);
+      if (this.props.model[key].render) {
+        return this.props.model[key].render(this.props.data);
```

Comment:
Alright. Anyway I want to let you know I'd rather change this implementation to another one closer to material-ui. At the first implementation I wasn't sure about the best way to compose this kind of components but after a year it's pretty clear that this approach is not flexible enough in some cases.